src/webportal/src/app/job/job-view/fabric/task-attempt/task-attempt-list.jsx

```diff
@@ -0,0 +1,626 @@
+        onRender: (item, idx) => {
+          return (
+            <div className={FontClassNames.mediumPlus}>{item.attemptId}</div>
+          );
+        },
+      },
+      {
+        key: 'taskAttemtState',
+        name: 'Task Attempt State',
+        minWidth: 120,
+        headerClassName: FontClassNames.medium,
+        isResizable: true,
+        onRender: item => (
+          <StatusBadge status={capitalize(item.attemptState)} />
+        ),
+      },
+      {
+        key: 'taskAttemptExitCode',
+        name: 'Task Attempt Exit Code',
+        minWidth: 230,
+        headerClassName: FontClassNames.medium,
+        isResizable: true,
+        onRender: (item, idx) => {
+          return isNil(item.containerExitSpec) ? (
+            <div className={FontClassNames.mediumPlus}>
+              {item.containerExitCode}
+            </div>
+          ) : (
+            <div className={FontClassNames.mediumPlus}>
+              {`${item.containerExitCode} (${item.containerExitSpec.phrase})`}
+            </div>
+          );
+        },
+      },
+      {
+        key: 'ip',
+        name: 'IP',
+        className: FontClassNames.mediumPlus,
+        headerClassName: FontClassNames.medium,
+        minWidth: 90,
+        maxWidth: 140,
+        isResizable: true,
+        fieldName: 'containerIp',
+        onRender: item => {
+          const ip = item.containerIp;
+          return !isNil(ip) && <div>{ip}</div>;
+        },
+      },
+      {
+        key: 'ports',
+        name: 'Ports',
+        className: FontClassNames.mediumPlus,
+        headerClassName: FontClassNames.medium,
+        minWidth: 150,
+        maxWidth: 300,
+        isResizable: true,
+        onRender: item => {
+          const ports = item.containerPorts;
+          return (
+            !isNil(ports) && (
+              <div className={c(t.truncate)}>
+                {flatten(
+                  Object.entries(ports).map(([key, val], idx) => [
+                    idx !== 0 && (
+                      <span className={t.ml2} key={`gap-${idx}`}></span>
+                    ),
+                    `${key}: ${val}`,
+                  ]),
+                )}
+              </div>
+            )
+          );
+        },
+      },
+      {
+        key: 'taskAttemptLaunchTime',
+        name: 'Task Attempt Launch Time',
+        headerClassName: FontClassNames.medium,
+        minWidth: 180,
+        maxWidth: 200,
+        isResizable: true,
+        onRender: item => {
+          return (
+            <div className={c(FontClassNames.mediumPlus)}>
+              {isNil(item.currentAttemptLaunchedTime)
+                ? 'N/A'
+                : DateTime.fromMillis(
+                    item.currentAttemptLaunchedTime,
+                  ).toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS)}
+            </div>
+          );
+        },
+      },
+      {
+        key: 'taskAttemptDuration',
+        name: 'Task Attempt Duration',
+        minWidth: 150,
```

Comment:
Align with others, and remove prefix, so just 'Duration'?

Reply:
done